crates/proof-of-sql/Cargo.toml

```diff
@@ -83,7 +83,7 @@ workspace = true
 
 [[example]]
 name = "hello_world"
-required-features = [ "blitzar", "test" ]
+required-features = [ "test" ]
```

Comment:
I think it should be possible to remove even the "test" required feature. I could be wrong, so if not, I'd be curious to know why. (In which case, I will probably want to fix that.)

Reply:
When the test feature is removed, Getting errors on the usage of the test_rand() function, TestAccessor, etc.. which uses the test feature 

![image](https://github.com/user-attachments/assets/aae11307-5367-4eaf-9830-7790f7bccbd9)

Reply:
Makes sense. That's fine, then.